> Assuming we test thunderbird builds.

Meh...

```bash
 > error[E0599]: no method named `setAlignment_` found for struct `objc::StrongRef<NSMutableParagraphStyle>` in the current scope
       >     --> toolkit/crashreporter/client/app/src/ui/macos/mod.rs:1064:24
       >      |
       > 1064 |                     ps.setAlignment_(cocoa::NSTextAlignmentRight);
       >      |                        ^^^^^^^^^^^^^
       >      |
       >     ::: toolkit/crashreporter/client/app/src/ui/macos/objc.rs:68:1
```

--- a/pkgs/applications/networking/browsers/firefox/common.nix
+++ b/pkgs/applications/networking/browsers/firefox/common.nix
@@ -98,6 +98,7 @@ in
 
   # Darwin
   apple-sdk_14,
+  apple-sdk_15,
   cups,
   rsync, # used when preparing .app directory
 
@@ -578,7 +579,7 @@ buildStdenv.mkDerivation {
       zip
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      apple-sdk_14
+      (if lib.versionAtLeast version "138" then apple-sdk_15 else apple-sdk_14)
       cups
     ]
     ++ (lib.optionals (!stdenv.hostPlatform.isDarwin) (

